src/controllers/itemDetails/index.html

```diff
@@ -187,6 +187,18 @@ <h2 class="sectionTitle sectionTitle-cards padded-right"></h2>
                     </div>
                 </div>
 
+                <div id="lyricsSection" class="verticalSection-extrabottompadding detailVerticalSection lyricsContainer hide">
+                    <h2 class="sectionTitle sectionTitle-cards padded-right">${Lyrics}</h2>
+                    <div is="emby-itemscontainer" class="vertical-list focuscontainer-x itemsContainer"></div>
```

Comment:
Ok right, I think focuscontainer-x must have been a remnant from which ever section I decided to copy for a skeleton. I never managed to work out what it did, So kept it in in case it was important. I will remove it.

Reply:
Fixed on [063f16a](https://github.com/jellyfin/jellyfin-web/pull/4733/commits/063f16aaae784a19bb4dd7606ae066da89818df3)